doc/user/content/quickstarts/application.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Build a real-time application"
+description: "  "
+menu:
+    main:
+        parent: quickstarts
+weight: 20
+aliases:
+  - /demos/real-time-application
+---
+
+**tl;dr** Materialize unleashes real-time powers for your applications. Process, serve, and display information quickly as a blink.
+
+{{< figure
+    src="https://user-images.githubusercontent.com/11491779/166932582-e5a9fd47-e397-4419-b221-e8f38c6f06f5.mp4"
```

Comment:
Can you see this rendered? It comes up as a broken link for me locally.

Reply:
It seems that it is not available anymore. Is there any commonplace to store content? I was thinking about using the WordPress storage.

Reply:
I can access it through the link, just not see it rendering on the docs website. 🤔  I also use the GitHub servers to store content, but Wordpress could be an option. What we could also do is make it a short GIF — there's no real need to have a 16s video for this. WDYT?

Reply:
I would love to! The options I've tried back in time turned it into a super pixelated monster, so I desisted

Reply:
Giphy usually works pretty well!

Reply:
I just created an account and tried with Giphy https://media.giphy.com/media/plAtuUgaZA90Jr7AzD/giphy.gif
Wdyt?